src/uu/mknod/src/parsemode.rs

```diff
@@ -4,19 +4,16 @@ use libc::{mode_t, S_IRGRP, S_IROTH, S_IRUSR, S_IWGRP, S_IWOTH, S_IWUSR};
 
 use uucore::mode;
 
-pub fn parse_mode(mode: Option<String>) -> Result<mode_t, String> {
-    let fperm = S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP | S_IROTH | S_IWOTH;
-    if let Some(mode) = mode {
-        let arr: &[char] = &['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
-        let result = if mode.contains(arr) {
-            mode::parse_numeric(fperm as u32, mode.as_str())
-        } else {
-            mode::parse_symbolic(fperm as u32, mode.as_str(), true)
-        };
-        result.map(|mode| mode as mode_t)
+pub const MODE_RW_UGO: mode_t = S_IRUSR | S_IWUSR | S_IRGRP | S_IWGRP | S_IROTH | S_IWOTH;
+
+pub fn parse_mode(mode: &str) -> Result<mode_t, String> {
```

Comment:
this file has been moved into uucore
by b1fcb621a88e85830c074a9ac5c52def1ef73244
could you please see how to port your change? sorry

Reply:
Done